tests/integration/drivers/vector/test_astra_db_vector_store_driver.py

```diff
@@ -0,0 +1,228 @@
+from __future__ import annotations
+
+import json
+import math
+import os
+
+import pytest
+from attrs import define, field
+
+from griptape.drivers import AstraDBVectorStoreDriver, BaseEmbeddingDriver, BaseVectorStoreDriver
+from tests.mocks.mock_tokenizer import MockTokenizer
+
+TEST_COLLECTION_NAME = "gt_int_test"
+TEST_COLLECTION_NAME_METRIC = "gt_int_test_dot"
+
+
+def astrapy_available() -> bool:
+    try:
+        return True
+    except Exception:
+        return False
+
+
+def astra_db_available() -> bool:
+    return all(
+        [
+            "ASTRA_DB_APPLICATION_TOKEN" in os.environ,
+            "ASTRA_DB_API_ENDPOINT" in os.environ,
+        ]
+    )
```

Comment:
Can remove. Our integration tests assume the `all` extra is installed.

Reply:
Should I remove the check on the env. vars as well (i.e. assume valid secrets are present)?
Or would I refine the `astra_db_available` to e.g. checking that the env.vars are not the empty string? (in the interest of not having test failing just because of no secrets passed).

Reply:
Yeah let's remove the env check as well. If running the full test suite we can assume that all env vars are set. Otherwise users can run individual test files.

Reply:
All right, thanks! (done).